doc/source/reference/routines.testing.rst

```diff
@@ -53,13 +57,16 @@ Decorators
    dec.skipif
    dec.slow
    decorate_methods
+   raises
```

Comment:
The docstring for `raises` has a warning that it should only be used with `nose`. I think we shouldn't index it.
```suggestion
```